bridge/src/lib.rs

```diff
@@ -41,9 +70,47 @@ methods!(
             connection.call(&rpc, request.clone())
         }, Some(|| {}));
 
-        let response = result.map_err(|e| {
-            VM::raise_ex(AnyException::new("Temporal::Bridge::Error", Some(&e.to_string())))
-        }).unwrap();
+        let response = result.map_err(|e| raise_bridge_exception(&e.to_string())).unwrap();
+
+        let enc = Encoding::find("ASCII-8BIT").unwrap();
+        RString::from_bytes(&response, &enc)
+    }
+
+    // TODO: Add telemetry configuration to this interface
+    fn init_telemetry() -> NilClass {
+        let telemetry_config = TelemetryOptionsBuilder::default()
+            .tracing_filter("temporal_sdk_core=DEBUG".to_string())
+            .logging(Logger::Console)
+            .build()
+            .map_err(|e| raise_bridge_exception(&e.to_string()))
+            .unwrap();
+
+        temporal_sdk_core::telemetry_init(&telemetry_config)
+            .expect("Unable to initialize telemetry");
+
+        NilClass::new()
+    }
+
+    fn create_worker(connection: AnyObject, namespace: RString, task_queue: RString) -> AnyObject {
+        let namespace = namespace.map_err(|e| VM::raise_ex(e)).unwrap().to_string();
```

Comment:
@Sushisource I haven't forgot about your suggestion to introduce a layer here to simplify error handling. Added a ticket for it — #26 and will implement separately